erizo/src/erizo/lib/Clock.h

```diff
@@ -0,0 +1,21 @@
+#ifndef ERIZO_SRC_ERIZO_LIB_CLOCK_H_
+#define ERIZO_SRC_ERIZO_LIB_CLOCK_H_
+
+#include <chrono>  // NOLINT
+
+namespace erizo {
+
+using clock = std::chrono::steady_clock;
+using time_point = std::chrono::steady_clock::time_point;
+using duration = std::chrono::steady_clock::duration;
+
+inline int64_t durationToMs(erizo::duration duration) {
```

Comment:
Consider putting this utility functions inside a class, even if they are still static. For instance, `ClockUtils::durationToMs(...)` would be more readable in my opinion